client/src/featureform/register.py

```diff
@@ -2582,7 +2582,11 @@ def get_model(self, name, local=False) -> Model:
         Returns:
             model (Model): Model
         """
-        model = get_model_info_local(name) if local else get_resource_info(self._stub, "model", name)
+        if local:
+            model = get_model_info_local(name)
+        else:
+            model_proto = get_resource_info(self._stub, "model", name)
+            model = Model(model_proto.name) if model_proto is not None else model_proto
```

Comment:
Should this return the model proto or fail?

Reply:
The pattern with the other resource get methods was to print out an message to the console in the event of a gRPC error. In these cases, the implicit return type is `None` so I tried to maintain that API here.

Reply:
I would just do `else None` for clarity for now. I blocked off time on my calendar this week to review that lingering PR that would remove this weird logic